.claude/skills/skill-ui-ux-design/SKILL.md

```
@@ -352,10 +378,19 @@ mcp__shadcn__search_items_in_registries({ query: "<component name>" })
-3. **Implementation readiness** — confirm specs are detailed enough for frontend-developer
-4. **Figma push-back** (if connected) — offer to push design tokens to Figma
+1. **Accessibility audit — run the checker, do not eyeball.** Every text/background pair in the token set goes through the contrast validator:
+
+```bash
+python3 "${HOME}/.claude-octopus/plugin/scripts/helpers/contrast-check.py" \
+  '<text-hex>:<bg-hex>' '<heading-hex>:<bg-hex>:large' '<muted-hex>:<bg-hex>' ...
+```
+
+Exit 1 means at least one pair fails WCAG AA — fix the palette and re-run before delivering. Include the checker output in the final document as evidence.
+
+2. **Slop check** — run the pre-ship checklist in `skills/blocks/design-taste.md`; two or more misses means revise before delivering
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)